server/apps/immich/src/utils/patch-open-api.util.ts

```diff
@@ -18,6 +19,10 @@ export function patchOpenAPI(document: OpenAPIObject) {
         continue;
       }
 
+      if ((operation.security || []).find((item) => !!item[Metadata.PUBLIC_SECURITY])) {
```

Comment:
What will this do? It would affect the auto-generated documentation site, right?

Reply:
Without this there was no way to un-add the security options that were set at the class level. This marks a specific route as "public" and deleted the security settings (added at the controller level). There are no open API spec changes in this PR, implying that it is generating the exact same document.